tests/test_dataset.py

```diff
@@ -146,3 +148,45 @@ def test_satellite_dataset_nan_to_num(sat_zarr_path):
 
     assert np.sum(X[:, :, 0, 0]) == -11 * 13
     assert np.sum(y[:, :, 0, 0]) == -11 * 24
+
+
+def test_validation_dataset(val_sat_zarr_path, val_dataset_hyperparams):
+    dataset = ValidationSatelliteDataset(
+        zarr_path=val_sat_zarr_path,
+        history_mins=60,
+        forecast_mins=180,
+        sample_freq_mins=15,
+    )
+
+    # There are 14949 init times which all models must make predictions for
+    assert len(dataset) == 14949
+
+    X, y = dataset[0]
+
+    # 11 channels
+    # 2 y-dim steps
+    # 1 x-dim steps
+    # (60 / 15) + 1 = 5 history steps
+    # (180 / 15) = 12 forecast steps
+    assert X.shape == (
+        11,
+        5,
+        val_dataset_hyperparams["y_geostationary_size"],
+        val_dataset_hyperparams["x_geostationary_size"],
+    )
+    assert y.shape == (
+        11,
+        12,
+        val_dataset_hyperparams["y_geostationary_size"],
+        val_dataset_hyperparams["x_geostationary_size"],
```

Comment:
Are these magic numbers specific to this test or more generic (e.g. all valid datasets will use them)? If generic, we should move them into constants in the package, if specific then it's fine to leave them here.